src/collect/collect.c

```diff
@@ -484,10 +489,12 @@ int main(int argc, char **argv)
         kickout();
 
         lseek(fd, 0, SEEK_SET);
-        ftruncate(fd, 0);
+        if (ftruncate(fd, 0)) { /* TODO: implement proper error handling */
+        }
         ret = missing(fd);
 
-        lockf(fd, F_ULOCK, 0);
+        if (lockf(fd, F_ULOCK, 0)) { /* TODO: implement proper error handling */
```

Comment:
it's okay to ignore this error via `(void) lockf(...);`, or even not unlock at all, since closing releases locks

Reply:
`(void)lockf(...);` does not work as expected:
````
collect.c: In function ‘main’:
collect.c:496:15: warning: ignoring return value of ‘lockf’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  496 |         (void)lockf(fd, F_ULOCK, 0);
      |               ^~~~~~~~~~~~~~~~~~~~~
````

I will keep the if and add a comment inside about it is safe to ignore.